src/Workspaces/Core/Portable/Remote/WellKnownSynchronizationKind.cs

```diff
@@ -22,6 +22,7 @@ internal enum WellKnownSynchronizationKind
     ProjectAttributes,
     DocumentAttributes,
     SourceGeneratedDocumentIdentity,
+    SourceGeneratorExecutionVersionMap,
```

Comment:
We need to be able to serialize over the map containing all the SG execution versions for all projects to the OOP side.  note: this is only done if OOP sees a top level checksum change for the version-map.